client/src/components/calendar/calendarStyle.css

```diff
@@ -0,0 +1,57 @@
+html,
+body,
+body > div {
+	/* the react root */
+	margin: 0;
+	padding: 0;
+	height: 100%;
+}
+
+h2 {
+	margin: 0;
+	font-size: 16px;
+}
+
+ul {
+	margin: 0;
+	padding: 0 0 0 1.5em;
+}
+
+li {
+	margin: 1.5em 0;
+	padding: 0;
+}
+
+b {
+	/* used for event dates/times */
+	margin-right: 3px;
+}
+
+.demo-app {
+	display: flex;
+	min-height: 100%;
+	font-family: Arial, Helvetica Neue, Helvetica, sans-serif;
+	font-size: 14px;
+}
+
+.demo-app-sidebar {
+	width: 300px;
+	line-height: 1.5;
+	background: #eaf9ff;
+	border-right: 1px solid #d3e2e8;
+}
+
+.demo-app-sidebar-section {
+	padding: 2em;
+}
+
+.demo-app-main {
+	flex-grow: 1;
+	padding: 3em;
+}
+
+.fc {
```

Comment:
fc is not meaningful. Anyway it's not used anywhere

Reply:
it's used by the library without our knowledge (is not in the code)